components/board/genericlinux/config.go

```diff
@@ -6,14 +6,12 @@ import (
 	"go.viam.com/rdk/components/board"
 	"go.viam.com/rdk/components/board/mcp3008helper"
 	"go.viam.com/rdk/resource"
-	"go.viam.com/rdk/utils"
 )
 
 // A Config describes the configuration of a board and all of its connected parts.
 type Config struct {
 	AnalogReaders     []mcp3008helper.MCP3008AnalogConfig `json:"analogs,omitempty"`
 	DigitalInterrupts []board.DigitalInterruptConfig      `json:"digital_interrupts,omitempty"`
-	Attributes        utils.AttributeMap                  `json:"attributes,omitempty"`
```

Comment:
can you check if the other board configs `pi`, `numato` or the `components/board/config.go` files have this unused attribute too?

Reply:
The Numato, fake, and pi boards all have this, but they all look unused. There is only 1 user in the database whose config defines this. I'll get rid of them all in the code, but not bother writing a config migration.